### DISCUSSION TITLE
Command won't run

### DISCUSSION THREAD

**Discussion by adgower** [tasmota/tasmotizer#130] [Q&A] [answered]
Sonoff s31 tasmota 12.0.2.1

https://tasmota.github.io/docs/Rules/#time-delayed-auto-off-switch
```
{"Rule1":{"State":"ON","Once":"OFF","StopOnError":"OFF","Length":97,"Free":414,"Rules":"on power#state=1 do backlog power %value%; ruletimer1 15 endon on rules@timer=1 do power on endon"}}
```

then

```
power on
```

results: in nothing...

**Comment by jziolkowski** [comment, marked as answer]
first of all this is not for tasmota support. Second, you have a typo in your second rule. it should be rules#timer=1, not @

**Reply by adgower** [reply]
first of all my bad lol. I read somewhere that discord was being moved to github discussions. I appreciate your patience. It's working now!

```
Rule1
  ON button1#state DO Backlog Power1 %value%; RuleTimer1 600 ENDON
  ON Rules#Timer=1 DO Power1 off ENDON

```
